src/styles/blocks/buy.scss

```diff
@@ -0,0 +1,19 @@
+.buy {
+  text-align: center;
+  text-decoration: none;
+  text-transform: uppercase;
+  padding: 12px 70px;
+  color: rgba(255, 255, 255, 1);
+  background-color: $blue-accent;
+  font-weight: 700;
+  font-size: 14px;
+  line-height: 16px;
+  border-radius: $rounding;
+  transition: $transition-duration;
+
+  &:hover {
+    background-color: $white;
+    color: $blue-accent;
+    border: $blue-border;
```

Comment:
When hovering the button, you can see that all blocks do a small "jump". The problem is that the button has no borders when not hovered, but gets borders added when you hover.
To fix this issue, you can pass this `border` property to the initial state of the class (without the hover).
Because the border is the same color of the background when not hovering, we won't notice it there, but doing so will prevent that "jump" from happening.
You could also use the same width border in the initial state, but with transparent color, if you wish